src/otx/algo/classification/efficientnet.py

```diff
@@ -265,9 +265,9 @@ def _build_model(self, head_config: dict) -> nn.Module:
         backbone = OTXEfficientNet(version=self.version, pretrained=self.pretrained)
         return ImageClassifier(
             backbone=backbone,
-            neck=GlobalAveragePooling(dim=2),
-            head=HierarchicalLinearClsHead(
-                in_channels=backbone.num_features,
+            neck=nn.Identity(),
+            head=HierarchicalCBAMClsHead(
+                in_channels=1280,
```

Comment:
```suggestion
                in_channels=backbone.num_features,
```

Reply:
Fix in [0f91b61](https://github.com/openvinotoolkit/training_extensions/pull/3810/commits/0f91b614d179a84d4476db57a761a6b2f11a8a7f)